### ISSUE TITLE
[BUG] 'Settings' object has no attribute 'LANGUAGE_COOKIE_HTTPONLY' with django-cms 3.10 and django 2.2

### ISSUE THREAD

**Issue by jdgoettsch** [django-cms/django-cms#7290]
## Description

Django-CMS 3.10 expects a LANGUAGE_COOKIE_HTTPONLY setting. [This setting is not part of Django in Django 2.2](https://docs.djangoproject.com/en/2.2/ref/settings/) but was [added in Django 3.0](https://docs.djangoproject.com/en/3.0/ref/settings/#language-cookie-httponly).

## Steps to reproduce

1. Install Django-CMS 3.10 with Django 2.2 LTS.

## Expected behaviour

Django-CMS should have a fallback value if LANGUAGE_COOKIE_HTTPONLY is missing.

## Actual behaviour

Django-CMS 3.10 expects the LANGUAGE_COOKIE_HTTPONLY setting, which is not part of Django 2.2, leading to an Internal Server Error.

## Additional information (CMS/Python/Django versions)

Django-CMS 3.10
Django 2.2 LTS
Python 3.8.x

```Traceback:

File "/usr/local/envs/[are.ucdavis.edu](http://are.ucdavis.edu/)-3.8/lib64/python3.8/site-packages/django/core/handlers/exception.py" in inner
 34.             response = get_response(request)

File "/usr/local/envs/[are.ucdavis.edu](http://are.ucdavis.edu/)-3.8/lib64/python3.8/site-packages/django/utils/deprecation.py" in __call__
 96.             response = self.process_response(request, response)

File "/usr/local/envs/[are.ucdavis.edu](http://are.ucdavis.edu/)-3.8/lib64/python3.8/site-packages/cms/middleware/language.py" in process_response
 26.             httponly=settings.LANGUAGE_COOKIE_HTTPONLY,

File "/usr/local/envs/[are.ucdavis.edu](http://are.ucdavis.edu/)-3.8/lib64/python3.8/site-packages/django/conf/__init__.py" in __getattr__
 80.         val = getattr(self._wrapped, name)

Exception Type: AttributeError at /
Exception Value: 'Settings' object has no attribute 'LANGUAGE_COOKIE_HTTPONLY'
Request information:
USER: AnonymousUser

GET: No GET data

POST: No POST data

FILES: No FILES data
```

## Do you want to help fix this issue?

* [ ] Yes, I want to help fix this issue and I will join #workgroup-pr-review on [Slack](https://www.django-cms.org/slack) to confirm with the community that a PR is welcome. 
* [X] No, I only want to report the issue.

**Comment by marksweb** [comment]
Thanks for raising @jdgoettsch - I wasn't aware they only came in with 3.0.

Related issues which the 3.10 middleware addressed;
https://github.com/django-cms/django-cms/issues/7092
https://github.com/django-cms/django-cms/issues/7106
https://github.com/django-cms/django-cms/issues/7107

**Comment by marksweb** [comment]
@jdgoettsch incase you're not aware, it's also worth noting that django 2.2 has now reached end-of-life. 3.2 is now the LTS.

**Comment by marksweb** [comment]
This should be resolved in the 3.10.1 release.

**Comment by mihalikv** [comment]
@marksweb This fix is not working because variable `DJANGO_3_0` means `less than or equal to Django 3.0`. So if I have Django 2.2.28 than it will try to import `settings.LANGUAGE_COOKIE_HTTPONLY`. It should be probably `if not DJANGO_2_2`.
